cdc/model/owner.go

```diff
@@ -203,8 +203,7 @@ func (w *TaskWorkload) Marshal() (string, error) {
 
 // TableReplicaInfo records the table replica info
 type TableReplicaInfo struct {
```

Comment:
can we also remove `TableReplicaInfo` ?

Reply:
It's still used by other structs, so remove it will make the other struct looks weird.
For example:
```go
type TaskStatus struct {
	Tables       map[TableID]*TableReplicaInfo `json:"tables"`
	Operation    map[TableID]*TableOperation   `json:"operation"`
	AdminJobType AdminJobType                  `json:"admin-job-type"`
	ModRevision  int64                         `json:"-"`
}
```